### DISCUSSION TITLE
Welcome to vaillant-arotherm-plus Discussions!

### DISCUSSION THREAD

**Discussion by cyberthom42** [cyberthom42/vaillant-arotherm-plus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tollentino** [comment]
Hi @all

bin jetzt auch hier. Wollen wir hier eine Verbrauchsdatenbank mit weiteren Eckdaten angeben, wie RT (SOLL), RT (IST) und AT (MW)?
Wie macht man das am besten?
Im Wiki-Bereich?
Als Excelfile?
Bin kein Entwickler und kenne Github nur schlecht aus Gaming/Modder Tagen.

LG

Tolentino

**Reply by Tollentino** [reply]
Habe gerade gesehen, dass wohl Projects das richtige Tool wäre? Da steht was von Feldern etc...

**Reply by Tollentino** [reply]
Ach sorry, ich hätte wohl in IDEAS posten sollen, sorry...

**Reply by Tollentino** [reply]
Habe jetzt mal hier ne Diskussion im passenden Bereich gestartet

[Verbrauchsdatenbank](https://github.com/cyberthom42/vaillant-arotherm-plus/discussions/14)

**Comment by cyberthom42** [comment]
Habe mal ein Projekt gestartet. Wir können ja zusammen damit "reifen" ;.)

**Reply by Tollentino** [reply]
hmm bei dir sehe ich keine Projects und meins kann ich nicht verlinken. S.o. neue Discussion.

**Comment by cyberthom42** [comment]
Ich habe dir zugriff (Collaborator) gegeben

**Comment by hasierughybvhl** [comment]
Hello zusammen, 
ein Ergänzungsvorschlag für die Teileliste wäre der Kodierwiderstand. Teilenummer VAILLANT 0020269259
Grüße aus OWL ;) 
Habe einfach auf die Tasten gehauen beim Anlegen des Benutzerkontos hier.

**Reply by hasierughybvhl** [reply]
Ah und das Dämmkit für den Kältemittelsammler.
Vaillant Wärmedämmung Set 0020278361
